core/src/main/java/org/openstack4j/openstack/provider/DefaultAPIProvider.java

```diff
@@ -171,6 +171,8 @@
 import org.openstack4j.api.trove.InstanceFlavorService;
 import org.openstack4j.api.trove.InstanceService;
 import org.openstack4j.api.trove.TroveService;
+import org.openstack4j.api.networking.TrunkService;
+
```

Comment:
These are unused. please remove

Reply:
I forgot to add the binding here that's why they are unused. I'll add in the next commit.